packages/marshal/src/rankOrder.js

```diff
@@ -142,16 +142,20 @@ export const makeComparatorKit = (compareRemotables = (_x, _y) => 0) => {
       case 'string': {
         // Within each of these passStyles, the rank ordering agrees with
         // JavaScript's relational operators `<` and `>`.
+        // @ts-expect-error FIXME narrowed
         if (left < right) {
           return -1;
         } else {
+          // @ts-expect-error FIXME narrowed
           assert(left > right);
           return 1;
         }
       }
       case 'symbol': {
         return comparator(
+          // @ts-expect-error FIXME narrowed
```

Comment:
I would not object to ditto if these stick around.
```suggestion
          // @ts-expect-error ditto, type narrowed
```